tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc

```diff
@@ -76,34 +77,50 @@ std::set<string> GetDeletedServables(
   return deleted_servables;
 }
 
-// Like PollFileSystemForConfig(), but for a single servable.
```

Comment:
Please document this function: what does it do? what do its arguments mean? (in particular, what does 'children' represent?)